app-text/poppler/poppler-20.11.0.recipe

```diff
@@ -294,6 +299,7 @@ REQUIRES_devel="
 	devel:libjpeg$secondaryArchSuffix
 	devel:liblcms2$secondaryArchSuffix
 	devel:libopenjp2$secondaryArchSuffix
+#	devel:libnss3$secondaryArchSuffix
```

Comment:
Why add this Line?

Reply:
That's one for @extrowerk :)

Reply:
AFAIR poppler could use libnss, the configure stage searching for it, but was unable to find it. Could be removed.